crates/astria-conductor/src/reader.rs

```diff
@@ -282,3 +281,27 @@ impl Reader {
         Ok(())
     }
 }
+
+fn public_key_to_bach32_address(key: &[u8]) -> Result<String> {
+    use sha2::{
+        digest::typenum::Unsigned as _,
+        Digest as _,
+    };
+    const ADDRESS_LENGTH: usize = 20;
```

Comment:
can use this also https://github.com/informalsystems/tendermint-rs/blob/32dad520d8996da0c0bf5b0fc8767fe3fd9fab8e/tendermint/src/account.rs#L19 this won't be changing though so doesn't really matter